concerned-coyotes/earlyinternet/earlyinternet/urls.py

```diff
@@ -1,21 +1,7 @@
-"""earlyinternet URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.0/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
```

Comment:
No need to remove this as its a nice overview of how to use or add urls

Reply:
Nonetheless, it is not required. So it is fine if it is removed.

I removed the redundant files / folders, so linting issues should be fixed.